lib/ansible/modules/system/authorized_key.py

```diff
@@ -227,6 +227,7 @@
 import shlex
 from operator import itemgetter
 
+from ansible.compat.six import PY3
```

Comment:
Thanks - I think this explains the `ImportError` that I was getting with my patch. I thought it was because I hadn't configured the Ansible environment correctly.